lib/foreman_inventory_upload.rb

```diff
@@ -70,8 +70,16 @@ def self.ensure_folder(folder)
     folder
   end
 
+  def self.inventory_base_url
+    ForemanRhCloud.base_url + "/api/inventory/v1/hosts"
```

Comment:
Is there an url helper for this?

Reply:
This is the URL helper :) It's cloud address, the full address would be similar to `https://cloud.redhat.com/api/inventory/v1/hosts`. It's not a local Foreman URL.